components/measurement/HeadMetadata.js

```diff
@@ -32,10 +32,17 @@ const HeadMetadata = ({
 
   return (
     <Head>
+      <title>
```

Comment:
Why can't we include date in the title also?

Reply:
Because otherwise it ends up being too long. The [recommended length for the title tag is less than 64 characters](https://www.w3.org/Provider/Style/TITLE.html)